code/common/analyticslib.q

```diff
@@ -96,8 +96,6 @@ intervals:{[args]
 
 /rack function
 rack:{[args]
-    / Call general check args function
-    checkargs[args;`table`keycols`base`fullexpansion`timeseries`round;" sSB"]; 
```

Comment:
PR description doesn't give much context - why is this being removed? are there any potential issues that could arise from not checking args?

Reply:
It seems the second arg to `checkarg[]` is setting the dict keys to all being required rather than the actual required keys ```` `table`keycols ```` from [here](https://aquaqanalytics.github.io/TorQ/analyticslib/#usage_3)
Looking at the other functions, they have specified required keys as well.

Reply:
Yes as Hamza has said above, the checkargs function is reporting the keys: 
\`table\`keycols\`base\`fullexpansion\`timeseries\`round 
as required, whereas this doesn't seem to be the intention from the [documentation](https://aquaqanalytics.github.io/TorQ/analyticslib/#usage_3) (should only be \`table and \`keycols required).  Removing this for now will allow use of the .al.rack function as intended.

Reply:
so why not keep table & keycols in here instead of completely removing? seems like if we remove this check, it will try to execute with those required args missing and then return a less clear error message when it later fails

Reply:
That's been re-added with those changes now